src/openrct2/GameState.cpp

```diff
@@ -318,7 +318,7 @@ namespace OpenRCT2
         }
 
         ResearchUpdate();
-        RideRatingsUpdateAll();
+        OpenRCT2::RideRating::UpdateAll();
```

Comment:
Qualifier shouldn't be needed, given this is already this is already in OpenRCT2 namespace.